yogstation/code/modules/clothing/shoes/colour.dm

```diff
@@ -1,3 +1,6 @@
+/obj/item/clothing/shoes/sneakers/white/Initialize()
+	. = ..()
+	LoadComponent(/datum/component/daniel)
+
 /obj/item/clothing/shoes/sneakers/white/pointed_at(var/mob/user)
-	spawn(10)
-		user.say("DAMN DANIEL!")
\ No newline at end of file
+	. = ..()
```

Comment:
Do you even need this proc?

Reply:
..() needs to be called to send the signal.  The . is just in case someone in some far future wasteland decides to add a return value to pointed_at().

Reply:
Yeah but doesn't it automatically call ``/atom/proc/pointed_at`` if you don't define this one?